data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -172,20 +172,24 @@ else
     IRONIC_HOST="${IRONIC_IP}"
 fi
 
+# Create a podman secret for the image-customization-server 
+sudo podman secret rm pull-secret || true
+base64 /root/.docker/config.json | sudo podman secret create pull-secret -
```

Comment:
is base64 command supposed to work without sudo? I'm asking because the file here is under /root

Reply:
Yep, the script is started by a service run by root